src/systems/ec/implementations/vanilla/packages/ec-style-typography/README.md

```diff
@@ -1,7 +1,86 @@
-# Typography
+# Paragraphs
 
-Be careful when applying the `.ecl-typography` class.
+<table class="ecl-table">
```

Comment:
where does this table come from?

Reply:
that was the previous documentation.
I will empty the file